app/assets/javascripts/discourse/app/lib/register-service-worker.js

```diff
@@ -1,10 +1,6 @@
 import getAbsoluteURL, { isAbsoluteURL } from "discourse-common/lib/get-url";
 
-export function registerServiceWorker(
-  container,
-  serviceWorkerURL,
-  registerOptions = {}
-) {
+export function registerServiceWorker(serviceWorkerURL, registerOptions = {}) {
```

Comment:
The `container` argument is unused for a while and this assumes this function is internal and ok to change